### DISCUSSION TITLE
Fluent-bit Setup for ECS Fargate

### DISCUSSION THREAD

**Discussion by vijaynauti** [fluent/fluent-bit#6710] [Q&A]
Hello, Trying to implement the Fluent-bit for sending logs to Cloudwatch and OpenSearch.  Application is hosted on AWS ECS Fargate based container.  Where with the help of awslogdriver it is sending logs to cloudwatch logs and as per the documentation it is sending STDOUT and STDERR.

Same way i want to configure fluent-bit for sending the logs in both the target(CW and ES). can someone please help how to achive it.

Also red so many documentation of Fluent-bit but didn't found any solution for using the tail in about situation. but docker container sending stdout and strerr logs but tail PLUGIN require PATH as parameter, Not sure how to use that.

**Comment by vijaynauti** [comment]
@PettitWesley

**Comment by PettitWesley** [comment]
Hello @vijaynauti 

We have these relevant examples:
- https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/send-to-multiple-destinations
- https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/config-file-type-file
- https://docs.fluentbit.io/manual/pipeline/outputs/opensearch
- https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch

Are all of your logs being produced to stdout & stderr? If so the above should be all you need. If you also need to tail a log file, this example should help: https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/ecs-log-collection
